erpnext_shopify/utils.py

```diff
@@ -167,9 +167,21 @@ def create_webhooks():
 
 def disable_shopify_sync(item):
 	"""Disable Item if not exist on shopify"""
+	frappe.db.rollback()
 	item.sync_with_shopify = 0
 	item.save()
 	frappe.db.commit()
 
 def get_shopify_item_image(shopify_id):
 	return get_request("/admin/products/{0}/images.json".format(shopify_id))["images"]
+
+def disable_shopify():
```

Comment:
misleading function names `disable_shopify_sync` and `disable_shopify`
Better names:
- `disable_shopify_sync_for_item`
- `disable_shopify_sync_on_exception`